tutorials/Tutorial For QA Registration.ipynb

```diff
@@ -0,0 +1,135 @@
+{
```

Comment:
in general, this tutorial is fine.
But it still feels like it could be fleshed out a bit, it's a little sparse:
- in a separate, well-labeled cell, put all the variables that need to be changed by the user
- spend a bit of space explaining what's actually happening and the context behind it